code/datums/traits/negative.dm

```diff
@@ -1,5 +1,87 @@
 //predominantly negative traits
 
+/datum/trait/depression
+	name = "Depression"
+	desc = "You sometimes just hate life."
+	mob_trait = TRAIT_DEPRESSION
+	value = -1
+	gain_text = "<span class='danger'>You start feeling depressed.</span>"
+	lose_text = "<span class='notice'>You no longer feel depressed.</span>" //if only it were that easy!
+	medical_record_text = "Patient has a severe mood disorder causing them to experience sudden moments of sadness."
+
+
+
+/datum/trait/family_heirloom
+	name = "Family Heirloom"
+	desc = "You are the current owner of an heirloom. passed down for generations. You have to keep it safe!"
+	value = -1
+	var/obj/item/heirloom
+	var/where_text
+	var/datum/component/mood/mood
+
+/datum/trait/family_heirloom/add()
+	mood = trait_holder.GetComponent(/datum/component/mood)
+
+/datum/trait/family_heirloom/on_spawn()
+	var/mob/living/carbon/human/H = trait_holder
+	var/obj/item/heirloom_type
+	if(SSevents.holidays && SSevents.holidays[APRIL_FOOLS]) //on april fools, pick from any item in the game
+		var/list/heirlooms = subtypesof(/obj/item)
+		for(var/V in heirlooms)
+			var/obj/item/I = V
+			if((!initial(I.icon_state)) || (!initial(I.item_state)) || (initial(I.flags_1) & ABSTRACT_1))
+				heirlooms -= V
+		heirloom_type = pick(heirlooms)
+	else
+		switch(trait_holder.mind.assigned_role)
+			if("Clown")
+				heirloom_type = /obj/item/bikehorn/golden
+			if("Mime")
+				heirloom_type = /obj/item/reagent_containers/food/snacks/baguette
+			if("Lawyer")
+				heirloom_type = /obj/item/gavelhammer
+			if("Janitor")
+				heirloom_type = /obj/item/mop
+			if("Security Officer")
+				heirloom_type = /obj/item/book/manual/wiki/security_space_law
+			if("Scientist")
+				heirloom_type = /obj/item/toy/plush/slimeplushie
+			if("Assistant")
+				heirloom_type = /obj/item/storage/toolbox/mechanical/old/heirloom
+	if(!heirloom_type)
+		heirloom_type = pick(
+		/obj/item/toy/cards/deck,
```

Comment:
Can I suggesy it be basically anything from the arcade machine?

Reply:
McUrist family pulse rifle